doc/tutorial/search/solution3.cpp

```diff
@@ -0,0 +1,26 @@
+#include <seqan3/io/stream/debug_stream.hpp>
+#include <seqan3/search/algorithm/search.hpp>
+#include <seqan3/std/view/subrange.hpp>
+
+using namespace seqan3;
+using namespace seqan3::search_cfg;
```

Comment:
I don't know, maybe *not* importing the search_cfg would make the config stuff easier to read? :thinking: dunno

Reply:
In this example it would be ok, but imagine:
```cpp
using namespace search_cfg;
configuration const cfg = max_error{total{2}, substitution{0}, insertion{1}, deletion{1}} | mode{strata{2}};
```

would be 

```cpp
configuration const cfg = search_cfg::max_error{search_cfg::total{2}, search_cfg::substitution{0}, search_cfg::insertion{1}, search_cfg::deletion{1}} | search_cfg::mode{search_cfg::strata{2}};
```

or not as bad

```cpp
using namespace search_cfg;
configuration const cfg = search_cfg::max_error{total{2}, substitution{0}, insertion{1}, deletion{1}} | search_cfg::mode{strata{2}};
```

Reply:
I agree that (2) doesn't look nice, but that's a design problem of the strong types. Maybe we should move those out of the sub-namespace.

In any case I really prefer not importing those namespaces, after all we introduced them exactly to distinguish their members from names in the general namespace. If we tell users to not differentiate we might as well put everything into seqan3::

Reply:
Ok, I'll make them explicitly.